src/main/java/bit/javaoop/AbstractEmployee.java

```diff
@@ -1,29 +1,39 @@
 package bit.javaoop;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public abstract class AbstractEmployee {
-    private String name;
-    private double salary;
+     private String name;
+    private Float salary;
+    private static List <AbstractEmployee> EmployeeList = new ArrayList<>();
```

Comment:
Uhh, why `static`? `static` fields are evil you should avoid them at all cost. They introduce something like a global state which maybe error prone and hard to test.